darts/tests/test_TCN.py

```diff
@@ -12,6 +13,10 @@ class TCNModelTestCase(unittest.TestCase):
     def setUpClass(cls):
         logging.disable(logging.CRITICAL)
 
+    @classmethod
```

Comment:
Why remove this directory only in TCN test case?

Reply:
I checked other test cases and the ones that use model that creates this directory already cleanup after themselves.

Reply:
Yeah I forgot to add this, thanks @Kostiiii!

Reply:
Approved